cmd/mdatagen/testdata/config_gen/expected_golang_output/batch

```diff
@@ -0,0 +1,63 @@
+// Code generated by github.com/atombender/go-jsonschema, DO NOT EDIT.
+
+package batchprocessor
+
+import "encoding/json"
+
+// Configuration parameters for the batch processor.
+type Config struct {
+	// MetadataCardinalityLimit corresponds to the JSON schema field
+	// "metadata_cardinality_limit".
+	MetadataCardinalityLimit int `json:"metadata_cardinality_limit,omitempty" yaml:"metadata_cardinality_limit,omitempty" mapstructure:"metadata_cardinality_limit,omitempty"`
+
+	// MetadataKeys corresponds to the JSON schema field "metadata_keys".
+	MetadataKeys []string `json:"metadata_keys,omitempty" yaml:"metadata_keys,omitempty" mapstructure:"metadata_keys,omitempty"`
+
+	// SendBatchMaxSize corresponds to the JSON schema field "send_batch_max_size".
+	SendBatchMaxSize int `json:"send_batch_max_size,omitempty" yaml:"send_batch_max_size,omitempty" mapstructure:"send_batch_max_size,omitempty"`
+
+	// SendBatchSize corresponds to the JSON schema field "send_batch_size".
+	SendBatchSize int `json:"send_batch_size,omitempty" yaml:"send_batch_size,omitempty" mapstructure:"send_batch_size,omitempty"`
```

Comment:
[go-jsonschema](https://github.com/omissis/go-jsonschema) doesn't seem to use `uint`. In theory, the `minimum: 0` in the schema would be a good enough workaround but it looks like it's not supported right now. 

Would you be happy if instances of `uint` are replaced with `int` + a validation that the number is `>= 0`?